packages/react-instantsearch/src/components/Highlight.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function Highlight({hit, attributeName, highlight}) {
+  const parsedHighlightedValue = highlight({hit, attributeName});
+  const reactHighlighted = parsedHighlightedValue.map((v, i) => {
+    if (!v.isHighlighted) return v.value;
+    const key = `split-${i}-${v.value}`;
+    return <em key={key} className="ais-highlighted__value">{v.value}</em>;
+  });
+  return <span className="ais-highlighted">{reactHighlighted}</span>;
```

Comment:
Small changes here. Since:
- users might want to style both the highlighted and non-highlighted values (like the google instant autocomplete), without having to go for complex CSS selectors (thus, add a class name)
- the widget is called Highlight

I propose we go for JSX that will output:

```html
<span class="ais-Highlight">
  <em class="ais-Highlight__highlighted>co</em>
  <span class="ais-Highlight__nonHighlighted">ca</span>
</span>
```

This way we are both consistent and we should answer most use cases of styling/targeting both highlighted and non-highlighted values. Even if the google instant example is a bit different (it's actually suggestions), I believe the use case should still be addressed because doable with Algolia.